### DISCUSSION TITLE
Trying to gauge Porter suitability for single-node non-swarm docker compose stack

### DISCUSSION THREAD

**Discussion by Steve973** [getporter/porter#1660] [Q&A] [answered]
Hello.  I have been trying to find documentation before coming to the forums, but I am having trouble determining if porter.sh is suitable for installing my single-node, non-swarm docker-compose service stack.  My initial deployment will be to take our team's application modules, as docker images, along with a directory structure for volumes, and containing resources that the containers need, and installing/deploying this onto a single server to host it.  So far, I have been using a bash script to handle this task, including setting up passwords and volumes.  This means that people who will be installing a deployment instance of our app stack need to know a good bit about the installation process.  Instead, I would prefer more of a turn-key installation and deployment.  If porter.sh is suitable for what I need to accomplish, what documentation will make this a bit easier to understand and digest for someone like me, who is brand new to CNAB?  How can I get started with learning about doing these things that I want to do?  If porter.sh is not really appropriate for what I want to do, and if it is acceptable here, please recommend a more appropriate solution.  Thanks in advance!

**Comment by vdice** [comment, marked as answer]
@Steve973 does the thread in https://github.com/getporter/porter/discussions/1665 help answer some of the questions here?

**Reply by Steve973** [reply]
Yes.  This was my first post here, and then I worked on the second one to be more specific, and I wasn't sure how to delete or close this one.
